articles/active-directory-b2c/page-layout.md

```diff
@@ -54,6 +65,15 @@ Page layout packages are periodically updated to include fixes and improvements
 
 ## Unified sign-in sign-up page with password reset link (unifiedssp)
 
+**2.1.1**
+- Allow picking DOM layout via policy or querystring parameter `pageFlavor`: classic, oceanBlue, slateGray.
+- Allow using password from iOS Keychain.
+- Added a UXString `heading` in addtion to `intro` to show on page as a title, it will be hidden by default.  
```

Comment:
"Added a UXString heading in addtion to intro to show on page as a title. This will be hidden by default."